workspace.go

```diff
@@ -131,10 +134,19 @@ type Workspace struct {
 	RunsCount                  int                   `jsonapi:"attr,workspace-kpis-runs-count"`
 
 	// Relations
-	AgentPool    *AgentPool    `jsonapi:"relation,agent-pool"`
-	CurrentRun   *Run          `jsonapi:"relation,current-run"`
-	Organization *Organization `jsonapi:"relation,organization"`
-	SSHKey       *SSHKey       `jsonapi:"relation,ssh-key"`
+	AgentPool    *AgentPool          `jsonapi:"relation,agent-pool"`
+	CurrentRun   *Run                `jsonapi:"relation,current-run"`
+	Organization *Organization       `jsonapi:"relation,organization"`
+	SSHKey       *SSHKey             `jsonapi:"relation,ssh-key"`
+	Outputs      []*WorkspaceOutputs `jsonapi:"relation,outputs"`
+}
+
+type WorkspaceOutputs struct {
+	ID        string      `jsonapi:"primary,workspace-outputs"`
```

Comment:
This is required as opposed to using the `StateVersionOutputs` because the primary ID here is `workspace-outputs` while the other is `state-version-outputs`

Reply:
* What do you think about using an embedded struct, so it really is more of an alias of the same thing?
* I'm noticing that it looks like workspace outputs attribute names differ ever so slightly from SVO names in the API? 😭

Hastily glancing I think it'd be:

```golang
type WorkspaceOutput struct {
	ID string                `jsonapi:"primary,workspace-outputs"`
	Type      string      `jsonapi:"attr,output-type"`
	StateVersionOutput
}
```

Reply:
Yeah already I tried to embed the struct, but something about how it unmarshals doesn't work. 

Here is an example: https://play.golang.org/p/63kdiT043q0

If you run it, you will see that name/type don't get unmarshaled. But if you comment `Bar` and uncomment the two other fields, it unmarshals correctly. I didn't dig into the exact reason, but I imagine it probably has to do with how jsonapi is looking at the field tags. I think its worth an independent look at that.

Reply:
Ah bummer, alrighty. That'd be neat to fix.